### ISSUE TITLE
Reduced redundancy in Landsat data access

### ISSUE THREAD

**Issue by kaulfusa** [NASA-IMPACT/veda-black-marble#3]
The algorithm currently downloads all the QA files each time a band is downloaded. Since the algorithm requires 3 bands, 3 copies of the QA files are downloaded. As the QA files apply to all bands, this operation needs to occur 1 time. 

Acceptance Criteria
- [ ] Landsat data download function pulls the 3 QA files only 1 time
- [ ] Open PR for review

**Comment by DImuthuUpe** [comment]
@kaulfusa Logic checks whether the files are already downloaded into the corresponding paths. If the file is there, no download is initialized. 
https://github.com/NASA-IMPACT/veda-black-marble/blob/main/docker/bm.py#L637

**Comment by kaulfusa** [comment]
Sorry, my comment needed to be 1 step downstream. The workflow creates a merged *A.TIFF, *.P.TIFF, and a *.R.TIFF for each band (so 3 in this case) when in reality these are the same merged files (i.e. B5_Merged_1A.TIFF, B4_Merged_1A.TIFF, and B3_Merged_1A.TIFF are all the same.) It shouldn't be necessary to have 3 copies.

**Comment by DImuthuUpe** [comment]
The updated notebook with a utility script was uploaded into Veda docs fork. It handles the issue mentioned above. We need to decide whether we want to keep the code in both VEDA docs and here

https://github.com/DImuthuUpe/veda-docs/tree/main/notebooks/tutorials/black_marble_generation

**Comment by kaulfusa** [comment]
Correct. At least until we have an open license attached to it. I'd also think we probably want this available only to a small group rather than available by default to all hub users.

**Comment by DImuthuUpe** [comment]
Added the code and notebook into this repository. https://github.com/NASA-IMPACT/veda-black-marble/pull/5
